qiskit/aqua/components/variational_forms/ry.py

```diff
@@ -35,7 +35,7 @@ class RY(VariationalForm):
                 'depth': {
                     'type': 'integer',
                     'default': 3,
-                    'minimum': 1
+                    'minimum': 0
```

Comment:
?

Reply:
depth 0 == only first layer of parametrized single qubit rotations, i.e. no entanglement. This does not achieve quantum advantage, but it needs to be available e.g. to benchmark and compare performance with and without entanglement.